src/framework/components/anim/component-binder.js

```diff
@@ -67,36 +67,50 @@ class AnimComponentBinder extends DefaultAnimBinder {
     }
 
     resolve(path) {
-        var propertyLocation = AnimBinder.decode(path);
-
-        var entity = this._getEntityFromHierarchy(propertyLocation.entityPath);
-
-        if (!entity)
-            return null;
+        var target = this.targetCache[AnimBinder.encode(path.entityPath, path.component, path.propertyPath)];
```

Comment:
ditto here, cache `encode` result here and use it in one of the cases below

Reply:
👍